frappe/chat/doctype/chat_room/chat_room.py

```diff
@@ -10,13 +10,14 @@
 from frappe.chat 						 import authenticate
 from frappe.core.doctype.version.version import get_diff
 from frappe.chat.doctype.chat_message	 import chat_message
-from frappe.chat.util import (
+from frappe.chat.util                    import (
```

Comment:
why whitespace???

Reply:
It was there in all chat module `.py` files, so was trying to be consistent. Removed whitespace from all imports now.